automation_testing/tests/Components/Button/button.spec.ts

```diff
@@ -108,4 +108,12 @@ test.describe.parallel("Components - Button - Kitchen Sink", () => {
 		expect(await buttonPage.getColorFromElement(buttonPage.button.nth(58))).toBe(expectColor);
 		expect(await buttonPage.getColorFromElement(buttonPage.button.nth(59))).toBe(expectColor);
 	});
+
+	test.skip("Validate Button has darkerRed on Hover.", async () => {
+		const expectColor = theme.newColors.darkerRed["100"];
```

Comment:
For some weird reason the test is giving another color but the color did change in the website.